### PR TITLE
Add org.kde.pix

### PR DESCRIPTION
**Pull request by ognarb**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://invent.kde.org/maui/mauikit/-/issues/32
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). I'm the maintainer of kde.org website, does it count? :) 
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by ognarb** [comment]
bot, build org.kde.pix

**Comment by ognarb** [comment]
bot, build org.kde.pix

**Comment by ognarb** [comment]
bot, build org.kde.pix

**Comment by ognarb** [comment]
bot, build org.kde.pix

**Comment by ognarb** [comment]
bot, build org.kde.pix

File: org.kde.pix.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    "id": "org.kde.pix",
+    "rename-icon": "pix",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "pix",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.pix.json:11]:
```suggestion
        "--socket=fallback-x11",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: appstream-change.patch [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+       </image>
+     </screenshot>
+   </screenshots>
++  <recommends>
++    <control>pointing</control>
++    <control>keyboard</control>
++    <control>touch</control>
++  </recommends>
++  <content_rating type="oars-1.0" />
++  <categories>
```
Comment (bilelmoussaoui) [inline appstream-change.patch:43]: If the categories are defined in the desktop file, that should be enough. Both files are merged together later on

**Comment by bilelmoussaoui** [comment]
cc @flathub/kde

**Comment by tsdgeos** [comment]
Looks good to me.

Carl I'm assuming you'll fix the appstream upstream too, right?

**Comment by ognarb** [comment]
> Looks good to me.
> 
> Carl I'm assuming you'll fix the appstream upstream too, right?

I already fixed it upstream https://invent.kde.org/maui/pix/-/commits/master and Camillo started fixing other Maui apps (see https://invent.kde.org/maui/mauikit/-/issues/32).

File: appstream-change.patch (hunk 2 above)
Comment (ognarb) [reply appstream-change.patch:43]: This is good to know. I will remove it from the patch and from upstream. Thanks

Comment (bilelmoussaoui) [reply appstream-change.patch]: That happens by calling appstream-compose which caches the icons, screenshots & merges both informations on a single file that used by the stores later one :)

**Comment by bilelmoussaoui** [comment]
bot, build org.kde.pix

**Comment by travier** [comment]
👍🏻

**Pull request closed without merging**